clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -2728,6 +2728,23 @@ static void emitKernelNameType(QualType T, ASTContext &Ctx, raw_ostream &OS,
   emitWithoutAnonNamespaces(OS, T.getCanonicalType().getAsString(TypePolicy));
 }
 
+int SYCLIntegrationHeader::getCppVersion() {
```

Comment:
Doesn't clang already have similar stuff?
@erichkeane , don't you know?

Reply:
I don't think we have exactly this function, but we DO similar logic for the __cpluplus macro in the preprocessor.